demos/include/iot_demo_runner.h

```diff
@@ -46,10 +46,20 @@
     #endif
     #if defined( democonfigMQTT_ECHO_TASK_PRIORITY )
         #undef democonfigDEMO_PRIORITY
-        #define democonfigDEMO_PRIORITY     democonfigMQTT_ECHO_TASK_PRIORITY
+        #define democonfigDEMO_PRIORITY    democonfigMQTT_ECHO_TASK_PRIORITY
```

Comment:
Does this file need to be executable? Also I don't think this space was meant to have been deleted

Reply:
Resolved